src/extractor/guidance/turn_lane_matcher.cpp

```diff
@@ -102,29 +102,34 @@ bool isValidMatch(const TurnLaneType::Mask &tag, const TurnInstruction instructi
     return false;
 }
 
+double getMatchingQuality( const TurnLaneType::Mask tag, const ConnectedRoad &road)
+{
+    const constexpr double idealized_turn_angles[] = {0, 35, 90, 135, 180, 225, 270, 315};
+    const auto idealized_angle = idealized_turn_angles[getMatchingModifier(tag)];
```

Comment:
Can we assert on the size here?

Looking into this, the array here has 8 items, but `getMatchingModifier` returns 11 different items?